tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -802,7 +809,7 @@ void test_parse_errors() {
 }
 }  // namespace
 
-// [[Timeout, 15]]
+// [[Timeout, 40]]
```

Comment:
Can we split the slow part of this up?

Reply:
The "slow" parts of this test are the `test_bbh_time_dependent_factory()` tests which take up about ~90% of this test with these changes. The reason for this is the `QuaternionFunctionOfTime`. Every time it is called, it solves an ODE to get the quaternion, integrating from the last update time to the time requested. However, the "last update time" here is just the initial time. So every time we want the quaternion we are integrating from 0.0 to 7.8 (the last tested time for this test, just as an example) with a tolerance of 1.0e-12. Integrating this far only a couple times is still pretty quick, but I calculated that there are ~83,000 time-dep coord map evaluations in this whole test...so things add up and it gets pretty slow.

One thing I thought about doing was changing the times to check from `{0.0, 4.4, 7.8}` to something like `{0.0, 0.4, 1.1, 1.6}`. This should keep the test under 15 seconds. I was just worried about restricting the times tested to small times.

Reply:
Yeah, I think that's better.  When the test tolerances are ~1e-10 and the timescales are ~0.2, a time of 1.6 isn't really very small.